src/Symfony/Component/Intl/Resources/data/regions/meta.php

```diff
@@ -754,4 +754,588 @@
+        214 => 'DO',
+        '012' => 'DZ',
+        218 => 'EC',
+        233 => 'EE',
+        818 => 'EG',
+        732 => 'EH',
+        232 => 'ER',
+        724 => 'ES',
+        231 => 'ET',
+        967 => 'EU',
+        246 => 'FI',
+        242 => 'FJ',
+        238 => 'FK',
+        583 => 'FM',
+        234 => 'FO',
+        250 => 'FR',
+        266 => 'GA',
+        826 => 'GB',
+        308 => 'GD',
+        268 => 'GE',
+        254 => 'GF',
+        831 => 'GG',
+        288 => 'GH',
+        292 => 'GI',
+        304 => 'GL',
+        270 => 'GM',
+        324 => 'GN',
+        312 => 'GP',
+        226 => 'GQ',
+        300 => 'GR',
+        239 => 'GS',
+        320 => 'GT',
+        316 => 'GU',
+        624 => 'GW',
+        328 => 'GY',
+        344 => 'HK',
+        334 => 'HM',
+        340 => 'HN',
+        191 => 'HR',
+        332 => 'HT',
+        348 => 'HU',
+        360 => 'ID',
+        372 => 'IE',
+        376 => 'IL',
+        833 => 'IM',
+        356 => 'IN',
+        '086' => 'IO',
+        368 => 'IQ',
+        364 => 'IR',
+        352 => 'IS',
+        380 => 'IT',
+        832 => 'JE',
+        388 => 'JM',
+        400 => 'JO',
+        392 => 'JP',
+        404 => 'KE',
+        417 => 'KG',
+        116 => 'KH',
+        296 => 'KI',
+        174 => 'KM',
+        659 => 'KN',
+        408 => 'KP',
+        410 => 'KR',
+        414 => 'KW',
+        136 => 'KY',
+        398 => 'KZ',
+        418 => 'LA',
+        422 => 'LB',
+        662 => 'LC',
+        438 => 'LI',
+        144 => 'LK',
+        430 => 'LR',
+        426 => 'LS',
+        440 => 'LT',
+        442 => 'LU',
+        428 => 'LV',
+        434 => 'LY',
+        504 => 'MA',
+        492 => 'MC',
+        498 => 'MD',
+        499 => 'ME',
+        663 => 'MF',
+        450 => 'MG',
+        584 => 'MH',
+        807 => 'MK',
+        466 => 'ML',
+        104 => 'MM',
+        496 => 'MN',
+        446 => 'MO',
+        580 => 'MP',
+        474 => 'MQ',
+        478 => 'MR',
+        500 => 'MS',
+        470 => 'MT',
+        480 => 'MU',
+        462 => 'MV',
+        454 => 'MW',
+        484 => 'MX',
+        458 => 'MY',
+        508 => 'MZ',
+        516 => 'NA',
+        540 => 'NC',
+        562 => 'NE',
+        574 => 'NF',
+        566 => 'NG',
+        558 => 'NI',
+        528 => 'NL',
+        578 => 'NO',
+        524 => 'NP',
+        520 => 'NR',
+        570 => 'NU',
+        554 => 'NZ',
+        512 => 'OM',
+        591 => 'PA',
+        604 => 'PE',
+        258 => 'PF',
+        598 => 'PG',
+        608 => 'PH',
+        586 => 'PK',
+        616 => 'PL',
+        666 => 'PM',
+        612 => 'PN',
+        630 => 'PR',
+        275 => 'PS',
+        620 => 'PT',
+        585 => 'PW',
+        600 => 'PY',
+        634 => 'QA',
+        959 => 'QM',
+        960 => 'QN',
+        962 => 'QP',
+        963 => 'QQ',
+        964 => 'QR',
+        965 => 'QS',
+        966 => 'QT',
+        'QV',
```

Comment:
is this normal?

Reply:
I've forced all the array keys to be numeric strings, prefixed with an underscore, to stop PHP trying to convert string numbers such as '077' to (int) 77.